.github/workflows/libwasmer-build-shim-linux-arm64.yml

```diff
@@ -0,0 +1,44 @@
+name: libwasmer-build-shim-linux-arm64
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+  pull_request:
+
+jobs:
+  build:
+    name: Build shim library for Linux ARM64
+    runs-on: ubuntu-latest
```

Comment:
Unfortunately, we cannot easily cross-compile for Linux/ARM64 from the `macos-13-xlarge` runner. And we cannot easily use docker on that runner, either. Thus, we fallback to `ubuntu`.